flake8-comprehensions:

```
>>> import flake8_docstrings
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/flake8_docstrings.py", line 12, in <module>
    import pydocstyle as pep257
  File "/usr/lib/python3.10/site-packages/pydocstyle/__init__.py", line 4, in <module>
    from .checker import ConventionChecker, check
  File "/usr/lib/python3.10/site-packages/pydocstyle/checker.py", line 34, in <module>
    from .wordlists import IMPERATIVE_BLACKLIST, IMPERATIVE_VERBS, stem
  File "/usr/lib/python3.10/site-packages/pydocstyle/wordlists.py", line 11, in <module>
    import snowballstemmer
ModuleNotFoundError: No module named 'snowballstemmer'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/site-packages/flake8_docstrings.py", line 23, in <module>
    import pep257
ModuleNotFoundError: No module named 'pep257'
```

pydocstyle:

```
>>> import pydocstyle
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/site-packages/pydocstyle/__init__.py", line 4, in <module>
    from .checker import ConventionChecker, check
  File "/usr/lib/python3.10/site-packages/pydocstyle/checker.py", line 34, in <module>
    from .wordlists import IMPERATIVE_BLACKLIST, IMPERATIVE_VERBS, stem
  File "/usr/lib/python3.10/site-packages/pydocstyle/wordlists.py", line 11, in <module>
    import snowballstemmer
ModuleNotFoundError: No module named 'snowballstemmer'
```

--- a/lang-python/flake8-builtins/autobuild/defines
+++ b/lang-python/flake8-builtins/autobuild/defines
@@ -1,0 +1,9 @@
+PKGNAME=flake8-builtins
+PKGSEC=python
+PKGDEP="python-3 hatchling"
+BUILDDEP="setuptools"
+PKGDES="Check for python builtins being used as variables or parameters"
+
+ABTYPE=pep517
+ABHOST=noarch
+NOPYTHON2=1

--- a/lang-python/flake8-builtins/spec
+++ b/lang-python/flake8-builtins/spec
@@ -1,0 +1,4 @@
+VER=3.0.0
+SRCS="git::commit=tags/$VER::https://github.com/gforcada/flake8-builtins"
+CHKSUMS="SKIP"
+CHKUPDATE="anitya::id=85942"

--- a/lang-python/flake8-quotes/autobuild/defines
+++ b/lang-python/flake8-quotes/autobuild/defines
@@ -1,0 +1,9 @@
+PKGNAME=flake8-quotes
+PKGSEC=python
+PKGDEP="python-3"
+BUILDDEP="setuptools"
+PKGDES="Flake8 extension for checking quotes in python"
+
+ABTYPE=python
+ABHOST=noarch
+NOPYTHON2=1

--- a/lang-python/flake8-quotes/spec
+++ b/lang-python/flake8-quotes/spec
@@ -1,0 +1,4 @@
+VER=3.4.0
+SRCS="git::commit=tags/$VER::https://github.com/zheller/flake8-quotes"
+CHKSUMS="SKIP"
+CHKUPDATE="anitya::id=17466"

--- a/lang-python/pydocstyle/autobuild/defines
+++ b/lang-python/pydocstyle/autobuild/defines
@@ -1,0 +1,9 @@
+PKGNAME=pydocstyle
+PKGSEC=python
+PKGDEP="python-3 poetry-core snowballstemmer"
+BUILDDEP="setuptools"
+PKGDES="Docstring style checker"
+
+ABTYPE=pep517
+ABHOST=noarch
+NOPYTHON2=1

--- a/lang-python/pydocstyle/spec
+++ b/lang-python/pydocstyle/spec
@@ -1,0 +1,4 @@
+VER=6.3.0
+SRCS="git::commit=tags/$VER;copy-repo=true::https://github.com/PyCQA/pydocstyle"
+CHKSUMS="SKIP"
+CHKUPDATE="anitya::id=15553"

